src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -484,9 +542,103 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
             case ToolbarItemName.RemoveAllRegions:
                 this.canvas.current.confirmRemoveAllRegions();
                 break;
+            case ToolbarItemName.ActiveLearning:
+                await this.predict();
+                break;
+        }
+    }
+
+    private predict = async () => {
```

Comment:
This code is very generic and should be moved out into a common location where you can pass in a `canvas` element and ultimately return an array of `IRegion` that can then get added to the canvas.

Reply:
Done but it receives an HTMLImageElement, not a canvas